tasks/validate.yml

```diff
@@ -1,4 +1,25 @@
 ---
+- name: Check ansible_env for required environment variables
+  ansible.builtin.set_fact:
+    account_id_is_set: "{{ ansible_env.NEW_RELIC_ACCOUNT_ID is defined }}"
+    api_key_is_set: "{{ ansible_env.NEW_RELIC_API_KEY is defined }}"
+
+- name: Check environment for required environment variables
+  when: not (account_id_is_set and api_key_is_set)
+  loop: "{{ environment }}"
+  ansible.builtin.set_fact:
+    account_id_is_set: "{{ item.NEW_RELIC_ACCOUNT_ID is defined }}"
+    api_key_is_set: "{{ item.NEW_RELIC_API_KEY is defined }}"
+  until: account_id_is_set and api_key_is_set
+  retries: 0
+  no_log: true
+
+- name: Assert required environment variables are set
+  ansible.builtin.assert:
```

Comment:
@rthorn-nr good work on the validation, but we also need a friendly user message telling the user what they need to provide and where. For example, "a NEW_RELIC_API_KEY is required, please provide this input as an environment variable"

Reply:
Good call, resolved by [7da3d60](https://github.com/newrelic/ansible-install/pull/17/commits/7da3d602666eff216370113c4bfa480df699b997)